Deepalakshmi.B/RailsAppPostgres/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
```

Comment:
@Deepalakshmi20 Can you please add the possible routes will be generated from this route file?

CC: @InbasasiS